src/config.py

```
@@ -0,0 +1,139 @@
+    model: str
+    documentation_suffix: str = ".md"
+
+    def __init__(self):
+        user_cwd = Path(os.getenv("USER_CWD", os.getcwd()))
+        load_dotenv()
+        os.chdir(user_cwd)
+
+        args = self.parse_args()
+
+        provider = os.getenv("AI_PROVIDER", "")
+        model = os.getenv("AI_MODEL", "")
```

Ruff

Local variable `model` is assigned to but never used

Remove assignment to unused variable `model`

(F841)